pkgs/development/ocaml-modules/lem/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, ncurses
+, makeWrapper
+, ocamlbuild
+, findlib
+, ocaml
+, num
+, zarith
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lem";
+  version = "2022-12-10";
+
+  src = fetchFromGitHub {
+    owner = "rems-project";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ZQgcuIVRkJS0KtpzjbO4OPHGg6B0TadWA6XpRir30y8=";
+  };
+
+  minimalOCamlVersion = "4.07";
+
+  nativeBuildInputs = [ makeWrapper ocamlbuild findlib ocaml ];
+  buildInputs = [ ncurses num ];
+  propagatedBuildInputs = [ zarith ];
+
+  installFlags = [ "INSTALL_DIR=$(out)" ];
+
+  preInstall = ''
+    mkdir -p $OCAMLFIND_DESTDIR
+  '';
+  postInstall = ''
+    wrapProgram $out/bin/lem --set LEMLIB $out/share/lem/library
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/rems-project/lem";
+    description = "A tool for lightweight executable mathematics";
+    maintainers = with maintainers; [ genericnerdyusername ];
+    license = with licenses; [ bsd3 gpl2 ];
+  };
```

Comment:
The `platforms` attribute seems missing.

Reply:
I'm unable to test anything other than `x86_64-linux` without emulation, is it ok if that's the only platform I put there?